src/Common/src/System/Diagnostics/Debug.Unix.cs

```diff
@@ -8,108 +8,118 @@ namespace System.Diagnostics
 {
     // Intentionally excluding visibility so it defaults to internal except for
     // the one public version in System.Diagnostics.Debug which defines
-    // another version of this partial class with the public visibility 
+    // another version of this partial class with the public visibility
     static partial class Debug
     {
-        private static string NewLine => "\n"; 
+#if !MONO
```

Comment:
Debug.Windows.cs will still initialize NewLine the way it does in your code which is like:
```
        private static string NewLine => Windows.NewLine;
         // internal and not readonly so that the tests can swap this out.
        internal static IDebugLogger s_logger = Windows.s_logger;
```
except you will no longer need to have it #ifdef'ed. 

The primary goal is to try and eliminate as much of the #ifdef'ed code as we can.